### DISCUSSION TITLE
Links in Exported Markdown

### DISCUSSION THREAD

**Discussion by nickali** [outline/outline#1603] [Self Hosting] [answered]
Let's say I have the following: 

- Document foo (url: foo-123)
  - Nested Document bar (url: bar-456)
- Document baz (url: baz-789)

Let's also assume baz links to bar.

When all the documents are exported, the markdown link in baz to bar is something like /doc/bar-456, which only works in Outline. What's the easiest way to convert the /doc/bar-456 to point to the file in the exported folder? 

Programmatically I can go in and create a map containing files and file path and search through all the documents and do a search and replace. Should this actually be part of the export functionality?

**Comment by tommoor** [comment, marked as answer]
> Should this actually be part of the export functionality?

I think it makes sense to be an _optional_ part of the export, yep! If there was an api option to `rewriteLinks` then that could be exposed in the UI later. I wouldn't change the default behaviour as it it works well in some cases.

**Reply by nickali** [reply]
I agree. Is there a way to get this on the roadmap?

**Reply by tommoor** [reply]
I created an issue here: https://github.com/outline/outline/issues/1604

Someone in the community may tackle it, I can't make any promises around when the core team might get to this
